make/tasks/lint/lockfiles.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
```

Comment:
I'm ok with keeping this a shell script but for these more complicated scripts, I wonder if we should move to Swift. Our contributors usually _love_ Swift and it's the language they're most comfortable with, so maybe it'd be worth some initial investment.

Reply:
If those scripts were in the package, we could even unit test them (which we could with fourier) but even just having a Swift script by itself could be a benefit.